2023-11-03 10:00:12,908 [main.py         | 48] INFO: Start to run instance_10
2023-11-03 10:00:12,909 [simulate_api.py | 60] INFO: Start time of the simulator: 2023-11-03 00:00:00, time interval:  600.00
2023-11-03 10:00:12,918 [input_utils.py  | 47] INFO: Get 154 factories
2023-11-03 10:00:13,909 [input_utils.py  | 51] INFO: Get 23562 routes
2023-11-03 10:00:13,946 [input_utils.py  | 56] INFO: Get 5 vehicles
2023-11-03 10:00:13,958 [input_utils.py  | 60] INFO: Get 100 orders
2023-11-03 10:00:13,959 [simulate_api.py | 87] INFO: Initial position of V_1 is e2d5093fbe36431f8986ddb0e1c586be
2023-11-03 10:00:13,960 [simulate_api.py | 87] INFO: Initial position of V_2 is 7fe14b93f0f04ee7a994ef5b2c1fdb72
2023-11-03 10:00:13,960 [simulate_api.py | 87] INFO: Initial position of V_3 is fa366fc87a124d32926daa5bb093129f
2023-11-03 10:00:13,960 [simulate_api.py | 87] INFO: Initial position of V_4 is e47399648fa842b2b8f80094343d8091
2023-11-03 10:00:13,961 [simulate_api.py | 87] INFO: Initial position of V_5 is becb4f85393540b287e7329758b8d832
2023-11-03 10:00:13,961 [simulate_environment.py | 99] INFO: **************************************************
2023-11-03 10:00:13,962 [simulate_environment.py | 103] INFO: cur time: 2023-11-03 00:10:00, pre time: 2023-11-03 00:00:00
2023-11-03 10:00:13,962 [simulate_environment.py | 141] INFO: Start to update the input of 2023-11-03 00:10:00
2023-11-03 10:00:13,963 [simulate_environment.py | 168] INFO: Get 0 unallocated order items, 0 ongoing order items, 0 completed order items
2023-11-03 10:00:15,561 [simulate_environment.py | 254] INFO: 2023-11-03 00:10:00, Item 0010370001-1: state = 0 < 2, we can not finish the simulation
2023-11-03 10:00:15,562 [evaluator.py    | 62] INFO: Traveling Distance of Vehicle V_1 is  0.000, visited node list: 2
2023-11-03 10:00:15,562 [evaluator.py    | 62] INFO: Traveling Distance of Vehicle V_2 is  0.000, visited node list: 2
2023-11-03 10:00:15,563 [evaluator.py    | 62] INFO: Traveling Distance of Vehicle V_3 is  0.000, visited node list: 2
2023-11-03 10:00:15,564 [evaluator.py    | 62] INFO: Traveling Distance of Vehicle V_4 is  0.000, visited node list: 2
2023-11-03 10:00:15,564 [evaluator.py    | 62] INFO: Traveling Distance of Vehicle V_5 is  0.000, visited node list: 2
2023-11-03 10:00:15,565 [evaluator.py    | 34] INFO: Total distance:  0.000
2023-11-03 10:00:15,565 [main.py         | 55] ERROR: Failed to run simulator
2023-11-03 10:00:15,567 [main.py         | 56] ERROR: Error: too many values to unpack (expected 2), Traceback (most recent call last):
  File "C:\Users\Administrator\Desktop\logistic\Demo\xingtian\simulator\dpdp_competition\main.py", line 51, in <module>
    score = simulate(Configs.factory_info_file, Configs.route_info_file, instance)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\Desktop\logistic\Demo\xingtian\simulator\dpdp_competition\src\simulator\simulate_api.py", line 94, in simulate
    simulate_env.run()
  File "C:\Users\Administrator\Desktop\logistic\Demo\xingtian\simulator\dpdp_competition\src\simulator\simulate_environment.py", line 131, in run
    _,destination_dict = Evaluator.cal_distance(self.history, self.route_map, len(self.id_to_vehicle))
    ^^^^^^^^^^^^^^^^^^
ValueError: too many values to unpack (expected 2)

